probe/cobra.go

```diff
@@ -0,0 +1,35 @@
+package probe
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func CobraCommand() *cobra.Command {
+	var (
+		probeOptions Options
+	)
+
+	prb := &cobra.Command{
+		Use:   "probe",
+		Short: "Check the liveness or readiness of a locally-running server",
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			if !probeOptions.IsValid() {
+				return fmt.Errorf("some options are not valid")
+			}
+			if err := NewFileClient(&probeOptions).GetStatus(); err != nil {
+				return fmt.Errorf("fail on inspecting path %s: %v", probeOptions.Path, err)
+			}
+			fmt.Fprintf(cmd.OutOrStdout(), "OK")
+			return nil
+		},
+	}
+	prb.PersistentFlags().StringVar(&probeOptions.Path, "probe-path", "",
```

Comment:
I don't know what this means.  Can you rewrite the message so someone unfamiliar will be able to understand?  I realize you are just moving this code but perhaps now is the time to improve the message.